src/libuv/Native.Settings

```diff
@@ -37,11 +35,13 @@
     <ClCompile>
       <RuntimeLibrary>MultiThreadedDebug</RuntimeLibrary>
       <Optimization>Disabled</Optimization>
+      <GenerateDebugInformation>true</GenerateDebugInformation>
```

Comment:
Why was it moved to debug? The idea is to generate symbols for Release build which should be archived and can be used to debug dumps provided by the user (e.g. from dr watson) if needed .